conftest.py

```diff
@@ -48,6 +54,20 @@ def json(self):
     return _Client()
 
 
+@pytest.fixture(params=all_backends)
+def backend(request):
+    phone_verification_settings = copy.deepcopy(
+        test_settings.DJANGO_SETTINGS.get("PHONE_VERIFICATION")
+    )
+    phone_verification_settings["BACKEND"] = f"phone_verify.backends.{request.param}"
+    if (
+        request.param == "nexmo.NexmoSandboxBackend"
+        or request.param == "nexmo.NexmoBackend"
+    ):
+        phone_verification_settings["OPTIONS"]["KEY"] = "fake"
```

Comment:
Why is this condition needed specifically?

Reply:
Just because, the key `KEY` exist for `nexmo` APIs.